File: ruby/fizzbuzz.rb
#!/usr/bin/ruby

#for i in 1..100 do
#puts((i%15>0)?(i%5>0)?(i%3>0)?i:"Fizz":"Buzz":"FizzBuzz");
#end;

(1..100).each{|i| puts((i%15>0)?(i%5>0)?(i%3>0)?i:"Fizz":"Buzz":"FizzBuzz")};

File: c/fizzbuzz.c
#include <stdio.h>
#include <stdlib.h>

int main(void)
{
    int i;
    char str[4];
    for(i=1; i<101; i++){
       sprintf(str,"%d",i);
       printf("%s\n",i%15?i%5?i%3?str:"Fizz":"Buzz":"FizzBuzz");
    }

    return EXIT_SUCCESS;
}
